app/assets/stylesheets/shared.scss

```diff
@@ -11,7 +11,7 @@
   #progressBar {
     color: #fff;
     width: 100%;
-    height: calc(48px + 1vh);
+    height: calc(43px + 1vh);
```

Comment:
@morfenza I would advice you to either define a custom variable (e.g. --progress-bar-height: xx ) here or use of the properties available on `stylesheets/config/variables` and apply them in all of these instances

Plus, I think you might want to drop the `+1vh` and calc function if you want to keep the same height

Reply:
Nice suggestion, I'll try to implement it! Thanks! <3